docs/source/concept_models/vivarium_iv_iron/concept_model.rst

```diff
@@ -396,6 +404,10 @@ Locations of interest to this project:
 
   We should confirm with the BMGF that they would like to model the GBD definition rather than standard definition from other data sources. 
 
+.. note::
+
+  The overall fertility rate among women of reproductive age is 0.055 for South Asia and 0.105 for Sub-Saharan Africa (not including stillbirths). Therefore, approximately these fractions of the total population of WRA will enter the population of interest of PLW in our simulation. 
```

Comment:
is this an annual crude birth rate?  if so i think value will be about 3x higher in our sim